Fix Wizard position in docs (#463)
OpenZeppelin/cairo-contracts: docs/modules/ROOT/pages/wizard.adoc

= Cairo Contracts Wizard
:page-notoc:

Not sure where to start? Use the interactive generator below to bootstrap your
contract and learn about the components offered in OpenZeppelin Cairo Contracts.

> CAUTION: If you haven’t yet, before moving forward, we strongly recommend checking the xref:extensibility.adoc[Extensibility Pattern] section to understand how we manage extensibility in a language lacking inheritance.

Reviewer: ```suggestion
NOTE: We strongly recommend checking the xref:extensibility.adoc[Extensibility Pattern] to understand how to extend from our library.
```
i don't think we need to use caution in here
